_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -27,6 +34,8 @@ the data.frame - that table that we started with that information from a CSV
 gets dumped into when we load it. In this lesson, we'll learn a few more things
 about working with data.frame.
 
+##Adding columns and rows in data.frame
```

Comment:
We generally follow the google R style guide for the SWC/DC R materials, so I suppose the google markdown style guide is also something to shoot for: https://github.com/google/styleguide/blob/gh-pages/docguide/style.md